admin/admin-model.php

```diff
@@ -257,15 +257,15 @@ public function update_language( $args ) {
 	}
 
 	/**
-	 * Validates data entered when creating or updating a language
+	 * Validates data entered when creating or updating a language.
 	 *
 	 * @see PLL_Admin_Model::add_language()
 	 *
 	 * @since 0.4
 	 *
-	 * @param array  $args
-	 * @param object $lang optional the language currently updated, the language is created if not set
-	 * @return bool true if success / false if failed
+	 * @param array  $args Parameters of {@see PLL_Admin_Model::add_language() or @see PLL_Admin_Model::update_language()}.
```

Comment:
The problem is: if we change only one doc block between `PLL_Admin_Model::add_language()` and `PLL_Admin_Model::update_language()`, which one should we trust?

Since we have a data structure to define languages, couldn't we pass `PLL_Language` instances to those three methods?

Reply:
We can't because this method is called *before* we create the language. `PLL_Language` handles languages already stored in DB.